odk1-src/collect-connect.rst

```diff
@@ -33,13 +33,34 @@ Configure server from QR code
 
 .. _other-collect-server-options:
 
+Configure server from settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Maybe add an introductory sentence? "If you don't have a configuration QR code, you can manually enter your server URL and optional credentials."